deployment/agent-deployment/ServiceGroupRoot/Scripts/pushAgentToAcr.sh

```diff
@@ -8,11 +8,20 @@ if [ -z $AGENT_IMAGE_TAG_SUFFIX ]; then
   exit 1
 fi
 
+#Make sure that tag being pushed will not overwrite an existing tag in mcr
+MCR_TAG_RESULT="`wget -qO- https://mcr.microsoft.com/v2/azuremonitor/containerinsights/ciprod/tags/list`"
```

Comment:
we should handle the case if the wget call failed, exit with appropriate error message.